apps/prairielearn/src/ee/pages/instructorAiGenerateDrafts/instructorAiGenerateDrafts.html.ts

```diff
@@ -89,10 +102,30 @@ export function InstructorAIGenerateDrafts({
                   )}
                 </tbody>
               </table>
+
+              ${DestroyQuestionsModal(resLocals)}
             </div>
           </div>
         </main>
       </body>
     </html>
   `.toString();
 }
+
+function DestroyQuestionsModal(resLocals) {
```

Comment:
Let's avoid using `resLocals` in child components if we can avoid it. For example:

https://github.com/PrairieLearn/PrairieLearn/blob/4b9b2ef7fa150e4e2615382e7d013626f368246c/apps/prairielearn/src/ee/pages/administratorInstitutionSaml/administratorInstitutionSaml.html.ts#L408

Reply:
Done in 16aa671758e5d243976b2f58594e0032923eb6f8.